Source/ACE.Server/WorldObjects/Hotspot.cs

```diff
@@ -152,32 +152,35 @@ private void Activate(Player player)
             var amount = DamageNext;
             var iAmount = (int)Math.Round(amount);
 
+            var pAmount = 0;
+
             switch (DamageType)
             {
                 default:
                     if (player.Invincible) return;
                     amount *= (float)player.GetLifeResistance(DamageType);
                     iAmount = (int)Math.Round(amount);
                     player.TakeDamage(this, DamageType, amount, Server.Entity.BodyPart.Foot);
+                    pAmount = Math.Abs(iAmount);
```

Comment:
iAmount is always positive here, so this statement is effectively pAmount = iAmount, ie. redundant